ghost/core/core/server/services/email-analytics/wrapper.js

```diff
@@ -0,0 +1,90 @@
+const logging = require('@tryghost/logging');
+const debug = require('@tryghost/debug')('jobs:email-analytics:fetch-latest');
+
+class EmailAnalyticsServiceWrapper {
+    init() {
+        if (this.service) {
+            return;
+        }
+
+        const {EmailAnalyticsService} = require('@tryghost/email-analytics-service');
+        const {EmailEventStorage, EmailEventProcessor} = require('@tryghost/email-service');
```

Comment:
We can move both classes to the `@tryghost/email-analytics-service` package in a future cleanup if needed